CHANGELOG.rst

```diff
@@ -16,6 +16,9 @@ something was changed
 [master]  (2020-**-**)
 ----------------------
 
+Changed
+*****
+- ``get_polarization_from_s1_filename()``: return type is now only ``str`` instead of ``Union[str, List[str]]``

```

Comment:
Please also mention the related issue as it has been done for other entries in the changelog.

Reply:
Done! Sorry, I forgot the issue reference :0